<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surgery Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>845</width>
      <height>580</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="qtopencv" name="tab">
     <attribute name="title">
      <string>Vision</string>
     </attribute>
    </widget>
    <widget class="OpenGLScene" name="tab_2">
     <attribute name="title">
      <string>Graphics</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionLoad_Video"/>
    <addaction name="actionOpen_Camera"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_Feature_Extraction"/>
    <addaction name="actionShow_Auto_Calibration"/>
    <addaction name="actionShow_3D_Estimation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt4"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image ...</string>
   </property>
  </action>
  <action name="actionLoad_Video">
   <property name="text">
    <string>Load Video ...</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera ...</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShow_Feature_Extraction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Feature Extraction</string>
   </property>
  </action>
  <action name="actionShow_Auto_Calibration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Auto-Calibration</string>
   </property>
  </action>
  <action name="actionShow_3D_Estimation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 3D Estimation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt4">
   <property name="text">
    <string>About Qt4</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtopencv</class>
   <extends>QWidget</extends>
   <header>qtopencv.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpenGLScene</class>
   <extends>QWidget</extends>
   <header>openglscene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
